lib/tracking/index.js

```diff
@@ -3,7 +3,7 @@ const https = require('https');
 const { default: axios } = require('axios');
 const crypto = require('crypto');
 
-const { logger } = require('probot/lib/logger');
+const { getLog } = require('probot/lib/helpers/get-log');
```

Comment:
This is an internal API. If you import any path other than just `probot`, I'd recommend to pin the version in `package.json`. 

Maybe you can use [`@probot/pino`](https://github.com/probot/pino/) directly?

Reply:
done